paging_test.py

```diff
@@ -3423,19 +3423,31 @@ def test_failure_threshold_deletions(self):
         supports_v5_protocol = self.supports_v5_protocol(self.cluster.version())
 
         self.fixture_dtest_setup.allow_log_errors = True
+        supports_guardrails = self.cluster.version() >= LooseVersion('4.0')
```

Comment:
it's a 3rd different way of checking this :)
I don't know where's the best place to add `supportsGuardrails` (perhaps the `Cluster` itself?), but we want it in one place

Reply:
Agreed 👍🏻